lecture2.tex

```diff
@@ -52,38 +52,53 @@ \section{Kombinatoriska bevis}
 
-  Vi studerar ett rutnät av $(n+1)\times(n+1)$ punkter, såsom i figuren.
+  Vi studerar ett rutnät av $(n+1)\times(n+1)$ punkter, såsom i figuren nedan.
 
   \begin{figure}
-    \includegraphics{graphics/counting_triangular_numbers_by_square.png}
-    \caption[][2cm]{Ett rutnät av punkter. Figur tagen direkt från förra årets föreläsningsanteckningar.}
+    \includegraphics{graphics/Rutnät av punkter.png}
+    \caption[][2cm]{Ett rutnät av $(n+1)\times(n+1)$ punkter. Bevis av summan av de först n talen.}
   \end{figure}
   
-  Vi ser på den nedre gröna triangeln, och försöker räkna antalet punkter i den. Vi ser att den vänstra kolumnen har $(n+1)-1 = n$ punkter, den näst vänstraste har $(n+1)-2=n-1$ punkter, och så vidare till den näst längst till höger som har en punkt, och kolumnen längst till höger har noll punkter i den gröna triangeln.
+  Vi ser på den nedre gröna triangeln, och försöker räkna antalet punkter i den. Vi ser att kolumnen längst till vänster i triangeln har $(n+1)-1 = n$ punkter, den näst längst till vänster har $(n+1)-2=n-1$ punkter, och så vidare till den näst längst till höger som har en punkt. Kolumnen längst till höger ligger utanför triangeln och har därför noll punkter i den gröna triangeln.
 
-  Alltså, om vi summerar över kolumnerna så får vi att det är totalt $n + (n-1) + \ldots + 2 + 1 = S(n)$ punkter i triangeln. Eftersom kvadraten så klart är helt symmetrisk är det lika många punkter i den övre gröna triangeln, och det är lätt att se att det är $n+1$ punkter på diagonalen.
+  Alltså, om vi summerar över kolumnerna så får vi att det är totalt $n + (n-1) + \ldots + 2 + 1 = S(n)$ punkter i triangeln. Eftersom kvadraten är helt symmetrisk så är det lika många punkter i den övre gröna triangeln, och det är lätt att se att det är $n+1$ punkter på diagonalen.
 
-  Alltså måste det totala antalet punkter i kvadraten vara $2S(n) + (n+1)$ -- men vi vet också, så klart, att det är $(n+1)^2$. Så om vi löser detta för $S(n)$ får vi att
+  Alltså måste det totala antalet punkter i kvadraten vara $2S(n) + (n+1)$ -- men vi kan också räkna det totala antalet punkter genom att multiplicera två av kvadratens sidor, dvs $(n+1)\times(n+1)=(n+1)^2$. Om vi löser detta för $S(n)$ får vi att
+    $$2S(n) + (n+1)=(n+1)^2$$
     $$S(n) = \frac{(n+1)^2 - (n+1)}{2} = \frac{(n+1)((n+1)-1)}{2} = \frac{n(n+1)}{2}$$
   precis som vi önskade.
+  
+  Denna bevismetod kan även användas för att summera andra aritmetiska talföljder, dvs talföljder $a_{1},\cdots,a_{n}$ på formen $a_{i+1}=a_{i}+k$ där k är en reell konstant. Vi skapar återigen ett rutnät som denna gång är en rektangel som delas in i exakt två parallelltrapetser. Se följande figur. 
+  
+  \begin{figure}[h]
+    \centering
+    \includegraphics[width=0.5\textwidth]{graphics/86C79F60-B920-4AA3-A6CB-DEA00A8C58A6.jpeg}
+    \caption{Rutnät av punkter. Bevis för summan av aritmetriska tlföljder. }
```

Comment:
talföljder*

(Lägger in småkommentarer som denna för att komma ihåg var jag skall redigera när jag lägger in ert arbete i huvudversionen av anteckningarna. Typos är så klart inte så särskilt signifikanta för den faktiska bedömningen.)